conf/application.conf

```diff
@@ -35,9 +36,38 @@ microservice {
       host = localhost
       port = 9250
     }
+
+    auth-login-api {
+      host = localhost
+      port = 8585
+    }
+
+    auth {
+      host = localhost
+      port = 8500
+      login-callback.base-url = "http://localhost:9063"
+      login-callback.path= "/pega-proof-of-concept/start"
+    }
+
+    company-auth {
+      url = "http://localhost:9025"
```

Comment:
are you still using this config? There might be some other bits of config that you can clean up as well. If you're not using this then the `sign-in-path` and `sign-out-path` don't technically belong in `company-auth` I don't think

Reply:
removed